server/hap/HapProcessCeilometer.cc

```diff
@@ -397,7 +397,7 @@ HatoholError HapProcessCeilometer::parseInstanceElement(
 	return HTERR_OK;
 }
 
-HatoholError HapProcessCeilometer::getAlarmList(void)
+HatoholError HapProcessCeilometer::getAlarmInfoTable(VariableItemTablePtr &trigTablePtr)
```

Comment:
There's no data strict named 'AlarmInfo'. So I feel 'Info' is a little redundant.